packages/calypso-ui/src/text-control/docs/example.jsx

```diff
@@ -0,0 +1,39 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { PureComponent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { TextControl } from '@automattic/calypso-ui';
```

Comment:
ah, this is why the self-dependency was added. Does `from '../'` work?

Reply:
Yeah it does 🤦‍♂. Thanks!